WcaOnRails/config/locales/en.yml

```
@@ -1886,6 +1880,32 @@ en:
   #context: User notifications
   notifications:
     connect_wca_id: "Connect your WCA ID to your account!"
+  payments:
+    #context: localized names of payment providers
+    payment_providers:
+      paypal: 'PayPal'
+      stripe: 'Stripe'
+    #context: Setting up and making payments
+    payment_setup:
+      account_connected: "You have successfully connected your %{provider} account and can now receive registration payments for this competition."
+      account_disconnected_success: "Sucessfully disconnected your %{provider} account."
+      account_disconnected_failure: "There was an error with disconnecting the %{provider} account."
+      account_not_connected: "There was a problem and we could not connect your %{provider} account."
+      disconnect_account: "Disconnect %{provider} Account"
+      confirm_account_disconnect: "Are you sure you want to disconnect the %{provider} account? This action cannot be undone."
+      supported_currency_warning: "Please make sure the currency you have selected is supported by your chosen payment integration! The supported currencies will depend on the country where the account account is based, you can check them below:"
```

Found 2 accounts in the sentence.
```suggestion
      supported_currency_warning: "Please make sure the currency you have selected is supported by your chosen payment integration! The supported currencies will depend on the country where the account is based, you can check them below:"
```

---

Good catch, thanks!